micromouse/AssertionHandler.h

```diff
@@ -29,7 +30,7 @@ inline static void logError( int code ) {
 inline static void printErrorCodes() {
```

Comment:
hmmm, beats the hell out of me, i thought i had to, does it need to be static? i would say no huh